docs/README.md

```diff
@@ -20,12 +20,24 @@ Temporal fixes these problems by:
 - Parsing a strictly specified string format
 - Supporting non-Gregorian calendars
 
+Temporal provides a number of separate types, in order to
+
+- Represent the information you have
+- Avoid buggy pattern of filling in 0 or UTC for missing info
+- Do appropriate calculations based on type
+
+
 ## Cookbook
 
 A cookbook to help you get started and learn the ins and outs of Temporal is available [here](./cookbook.md).
 
 ## API Documentation
 
+Types whose name starts with "Plain" (like `Temporal.PlainDate`, `Temporal.PlainTime`, and `Temporal.PlainDateTime`) do not, by convention, have an associated time zone.
+Converting between such types and exact time types (`Temporal.Instant` and `Temporal.ZonedDateTime`) can be ambiguous because of time zones and daylight saving time.
```

Comment:
Maybe we can give a hint of what to come, saying briefly something like, "and the Temporal API lets developers configure the resolution of this ambiguity".